StortSpelProjekt/Engine/src/Renderer/Renderer.cpp

```diff
@@ -498,13 +496,18 @@ void Renderer::InitDirectionalLightComponent(Entity* entity)
 {
 	component::DirectionalLightComponent* dlc = entity->GetComponent<component::DirectionalLightComponent>();
 	// Assign CBV from the lightPool
-	std::wstring resourceName = L"DirectionalLight_DefaultResource";
-	ConstantBuffer* cbd = m_pViewPool->GetFreeCBV(sizeof(DirectionalLight), resourceName);
+	std::wstring resourceName = L"DirectionalLight";
+	ConstantBuffer* cb = m_pViewPool->GetFreeCB(sizeof(DirectionalLight), resourceName);
 
 	// Check if the light is to cast shadows
 	SHADOW_RESOLUTION resolution = SHADOW_RESOLUTION::UNDEFINED;
```

Comment:
Since it's a enum value you are defining I would move this to if(shadowRes == -1) resolution = ________

Reply:
no